app/controllers/follows_controller.rb

```diff
@@ -5,8 +5,9 @@ class FollowsController < ApplicationController
   def create
```

Comment:
Consider removing the usage of `before_action :get_user` - the DRY benefit might be outweighed by the added complexity, it makes the actions a little less easy to understand.

Also consider removing the `if` check from:

```ruby
@user = User.find(params[:user_id]) if !params[:user_id].blank?
```